src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,26 @@
-export const TodoInfo = () => {};
+import { getUserById } from '../../App';
```

Comment:
Importing `getUserById` from `App.tsx` goes against the guideline of preparing data in the App component. Instead, consider fetching the user data in the App component and passing the user object as a prop to `TodoInfo`. This will ensure a cleaner and more manageable data flow.